docker-compose-test.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.8"
+services:
+  test-pgdb:
+    container_name: test-pgdb
+    image: postgres:15-alpine
+    ports:
+      - "5433:5433"
+    hostname: localhost
+    environment:
+      POSTGRES_USER: chingu
+      POSTGRES_PASSWORD: chingu
+      POSTGRES_DB: dashboard
```

Comment:
Should we have a different db name here than in the dev environment?

Reply:
I think it should be alright as is, as they'll be running on different container names with different `DATABASE_URL`s and use the same ports so couldn't run contemporaneously, anyway.